notebooks/community/ml_ops/stage3/get_started_with_bqml_pipeline_components.ipynb

```diff
@@ -1,1199 +1,1225 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #75.          })
</pre><p>This should be properly indented:</p><p> </p><p>         }</p><p>     )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>done</p>